src/yardstick/__init__.py

```diff
@@ -23,26 +35,26 @@ def compare_results(
         for result in results:
             result.matches = matches_filter(result.matches)
 
-    digests = set(r.config.image_digest for r in results)
+    digests = {r.config.image_digest for r in results}
     if len(digests) != 1:
         raise RuntimeError(f"image digests being compared do not match: {digests}")
 
     return comparison.ByPreservedMatch(results=results)
 
 
-def compare_results_against_labels(  # pylint: disable=too-many-arguments
+def compare_results_against_labels(  # noqa: PLR0913
     descriptions: list[str],
     result_set: Optional[str] = None,
     fuzzy: bool = False,
     year_max_limit: Optional[int] = None,
     year_from_cve_only: bool = False,
     label_entries: Optional[list[artifact.LabelEntry]] = None,
-    matches_filter: Optional[callable] = None,
+    matches_filter: Optional[Callable] = None,
     store_root: Optional[str] = None,
 ) -> tuple[
     list[artifact.ScanResult],
     list[artifact.LabelEntry],
-    dict[str, list[comparison.AgainstLabels]],
+    dict[str, comparison.AgainstLabels],
```

Comment:
I wanted to double check this because it seems to have changed type not just formatting. A list was never returned?

Reply:
that's correct, I made this change manually based on feedback from mypi -- since mypy checks were previously disabled this type hint has been wrong for a while.